HybridCar.hpp

```diff
@@ -1,19 +1,14 @@
+#pragma once
+
 #include "ElectricEngine.hpp"
 #include "PetrolEngine.hpp"
+#include "ElectricCar.hpp"
+#include "PetrolCar.hpp"
 
-class HybridCar
+class HybridCar : public virtual  ElectricCar, public virtual  PetrolCar 
```

Comment:
```suggestion
class HybridCar : public ElectricCar, public PetrolCar 
```
virtual is not needed here. Only ElectricCar and PetrolCar should inherit virtually.